.github/workflows/nightly.yml

```diff
@@ -11,6 +11,12 @@ permissions:
   # implicitly all other scopes not listed become none
 
 on:
+  workflow_dispatch:
+  push:
+    paths:
+      # Run if a pull request changes this workflow to
+      # validate it works properly before merging.
+      - '.github/workflows/nightly.yml'
```

Comment:
This doesn't seem to work. My suspicion is that this configuration takes effect only once it was merged. I.e. with the next change to `nightly.yml`. Though, admittedly that might be wishful thinking.

Reply:
I guess the easiest thing to do is just to merge this and then open a test PR that tries modifying it and see if it works.